components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
+                        />
+                    </l-control>
+                    <PriorityLayers :map="map" />
+                    <MonitoringLayers :map="map" />
+                    <SupportLayers />
+                    <AlertLayers :map="map" />
+                </l-map>
+            </client-only>
+        </v-col>
+        <v-col cols="4" class="pl-1">
+            <div
+                style="
+                    border-right: 0.5px solid gray;
+                    border-top: 0.5px solid gray;
+                    border-bottom: 0.5px solid gray;
+                    height: 100%;
+                "
             >
-                <l-control position="topleft">
-                    <div class="pa-1 map-action-buttons">
-                        <div
-                            v-if="user.settings.map_zoom_buttons_visible"
-                        ></div>
-
-                        <div class="div-spacer" />
+                <div class="d-flex justify-center align-center ma-2">
+                    <div style="width: 20%">
+                        <v-img contain class="mr-4" src="/img/funai.svg" />
                     </div>
-                </l-control>
-
-                <l-control
-                    class="leaflet-coordinates-control"
-                    position="bottomleft"
-                >
-                    <div v-if="user.settings.map_pointer_coordinates_visible">
-                        {{ cursorCoordinates.lat }},
-                        {{ cursorCoordinates.lng }}
+                    <div style="width: 25%">
+                        <v-img contain src="/img/logocmr_normal_min.png" />
                     </div>
-                </l-control>
-
-                <l-control-scale
-                    v-if="user.settings.map_scale_visible"
-                    position="bottomleft"
-                />
-
-                <l-geo-json
-                    ref="interestArea"
-                    :geojson="interestArea"
-                    :options-style="interestStyle"
-                    :visible="showInterestArea"
-                />
-
-                <SupportLayers />
-
-                <MonitoringLayers v-if="!monitoringGeoserver" :map="map" />
-
-                <BaseWmsMetadataPopup :map="map" />
-
-                <PriorityLayers :map="map" />
-            </l-map>
-
-            <div v-if="loading" class="loading-background">
+                </div>
+                <div class="text-center font-weight-bold">
+                    <p class="font-weight-bold mb-1">
+                        {{ titleMap }}
+                    </p>
+                    <p class="mb-2">
+                        Centro de Monitoramento Remoto - Fundação Nacional do
+                        Índio
+                    </p>
+                </div>
+                <div
+                    class="d-flex justify-center"
+                    style="height: 150px; max-height: 150px; width: 100%"
```

Comment:
inline style? Why dont you use vue style?

Reply:
Converted to class.